react/features/filmstrip/reducer.js

```diff
@@ -92,7 +94,26 @@ const DEFAULT_STATE = {
      * @public
      * @type {Set<string>}
      */
-    visibleRemoteParticipants: new Set()
+    visibleRemoteParticipants: new Set(),
+
+    /**
+     * The width of the resizable filmstrip.
+     *
+     * @public
+     * @type {Object}
+     */
+    width: {
+        /**
+         * Current width. Affected by: user filmstrip resize,
+         * window resize, panels open/ close.
+         */
+        current: null,
```

Comment:
I'm wondering why do you use `null` here and why not `undefined`? 

If you go with `undefined` you can even just do `width: {}`.

Reply:
I could do just `width: {}`, but I wanted to show the object structure and have comments for the properties. And I don't see any benefit of using `undefined` over `null`

Reply:
For me the main disadvantage is that null is an object and the typeof checks we may run into problems when doing typeof checks. Also the undefined is the default value for any variable and I just find it easier to maintain in the future. On other hand is there any benefit of using `null` over `undefined` in this case? I'm having a hard time to find any...

Note: I'm not saying you have a bug here and this is just my personal preference.

Reply:
Ok, I understand, I'll change them to undefined, but I'll leave them in with the comments

Reply:
Actually, we need it to be null, because on mount the setVertivalFilmstrip action is dispatched and that converts the initial values into number. Number(null) is 0 which is fine, but Number(undefined) is NaN which is not equal to anything and such keeps triggering the subscriber and causing infinite calls

Reply:
Could you please point me to this code?

Reply:
In subscriber.web.js there's a listener for width.current; this is called on mount adn this dispatches setVerticalViewDimensions; this action calls calculateThumbnailSizeForResizableVerticalView and in this function there's a Math.max, which tries to convert to number all the params